### PR TITLE
send release complete from script

### PR DESCRIPTION
**Pull request by Ximinhan**
Trying to migrate sendCImessage and send email in promote Jenkins file into promote python script
depends on https://github.com/openshift-eng/aos-cd-jobs/pull/4118

### CODE REVIEW

File: pyartcd/config.example.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ cc = []
 qe_notification_recipients_ocp4=["multi-arch-zstream-testing@redhat.com", "aos-qe@redhat.com"]
 qe_notification_recipients_ocp3=["wzheng@redhat.com", "gpei@redhat.com", "aos-qe@redhat.com", "mp-entitlement-qe@redhat.com", "openshift-lp-test@redhat.com", "pprakash@redhat.com"]
 promote_image_list_recipients = ["openshift-ccs@redhat.com"]
+promote_complete_recipients = ['aos-cicd@redhat.com', 'aos-qe@redhat.com', 'aos-art-automation+new-release@redhat.com']
```
Comment (vfreex) [inline pyartcd/config.example.toml:18]: This is an example config. The actual config file is at https://github.com/openshift-eng/aos-cd-jobs/blob/master/config/artcd.toml

Comment (Ximinhan) [reply pyartcd/config.example.toml:18]: updated in https://github.com/openshift-eng/aos-cd-jobs/pull/4118

File: pyartcd/pyartcd/pipelines/promote.py [hunk 2]
```diff
@@ -456,6 +456,9 @@ async def run(self):
 
         self.create_cincinnati_prs(assembly_type, data)
 
+        # send promote complete email
+        self.send_promote_complete_email(data["name"], release_infos)
```
Comment (thegreyd) [inline pyartcd/pyartcd/pipelines/promote.py:460]: Here can we have try / except -> send msg on slack and continue?

**Pull request merged**